main/integration-tests/support/setup.js

```diff
@@ -151,6 +160,10 @@ class Setup {
       userRole,
       projectId,
     });
+
+    // Delay to prevent eventual consistency issues from affecting the next call.
+    await sleep(1000);
```

Comment:
Doesn't the 'create' already introduce a delay?

Reply:
Fixed as explained above